art/urls.py

```diff
@@ -27,6 +27,7 @@
     path('jet/', include('jet.urls', 'jet')),
     path('jet/dashboard/', include('jet.dashboard.urls', 'jet-dashboard')),
     path('api-auth/', include('rest_framework.urls')),
+    path('o/', include('oauth2_provider.urls', namespace='oauth2_provider'))
```

Comment:
i'm thinking we move the last 2 urls to `api/urls.py` since they're for the api

Reply:
More for an organization thing yes ?

Reply:
yea.. once it's moved to `api/urls.py`

Reply:
`api-auth/` too

Reply:
Done.